tests/kallax.go

```diff
@@ -3120,29 +3162,32 @@ func (rs *PersonResultSet) Close() error {
 
 // NewPet returns a new instance of Pet.
 func NewPet(name string, kind string, owner *Person) (record *Pet) {
-	record = newPet(name, kind, owner)
-	if record != nil {
-		record.SetID(kallax.NewID())
-	}
-	return
+	return newPet(name, kind, owner)
```

Comment:
that's for the user to implement, because we don't know what they want as value in there